portal/resource_gallery.yaml

```diff
@@ -875,6 +878,43 @@
     - physical oceanography
     - data science
 
+- title: 'Environmental Data Science Book'
+  description: 'A living, open and community-driven resource to showcase and support computational
+    notebooks for collaborative, reproducible and transparent Environmental Science'
+  url: 'https://github.com/alan-turing-institute/environmental-ds-book'
```

Comment:
This link (which is what people find when they click on the gallery entry) is to the *source repo*.

Does it make more sense to link to the rendered book at https://the-environmental-ds-book.netlify.app/welcome.html ?

Reply:
I would suggest this link https://the-environmental-ds-book.netlify.app/ just in case they rename welcome to something else... but good point @brian-rose

Reply:
@acocac preference here?

Reply:
Hi! I've pointed to the GitHub resource instead of the website as we're in the discussion of getting a custom domain. If you think the website is better, please change it. I'll report any changes in the URL by opening a PR. Thank you.

Reply:
Ok sounds good! We will merge it as is. Please feel free to update with a new PR whenever you're ready!